packages/component-elife-app/server/routes/index.test.js

```diff
@@ -1,116 +1,29 @@
+  return supertest(app)
+}
 
+describe('ping route test', () => {
   it('returns success when app ok', async () => {
     const request = makeApp()
     await request.get('/ping').expect(200)
   })
-
-  describe('Response Errors', () => {
-    it('returns failure when an invalid response is returned from the database', async () => {
-      setDbSuccess(0)
-      const request = makeApp()
-      const response = await request.get('/ping').expect(410)
-      expect(response.body).toHaveLength(1)
-      expect(response.body[0]).toEqual('Database Error')
-    })
-
-    it('returns failure when an invalid response is returned from S3', async () => {
-      setS3Success(0)
-      health.checkDataBase.mockResolvedValue(17)
-      const request = makeApp()
-      const response = await request.get('/ping').expect(410)
-      expect(response.body).toHaveLength(1)
-      expect(response.body[0]).toEqual('S3 Error')
-    })
-
-    it('both S3 and database return invalid responses', async () => {
-      setS3Success(0)
-      setDbSuccess(0)
-      const request = makeApp()
-      const response = await request.get('/ping').expect(410)
-      expect(response.body).toHaveLength(2)
-      expect(response.body[0]).toEqual('Database Error')
-      expect(response.body[1]).toEqual('S3 Error')
-    })
-  })
-
-  describe('Connection Errors', () => {
-    it('returns connection error on S3', async () => {
-      setS3Error()
-      const request = makeApp()
-      const response = await request.get('/ping').expect(410)
-      expect(response.body).toHaveLength(1)
-      expect(response.body[0]).toEqual('S3 Error')
-    })
-
-    it('returns connection error on database', async () => {
-      setDbSuccess(new Error('connection Error'))
-      const request = makeApp()
-      const response = await request.get('/ping').expect(410)
-      expect(response.body).toHaveLength(1)
-      expect(response.body[0]).toEqual('Database Error')
-    })
-
-    it('both returns connection errors', async () => {
-      setS3Error()
-      setDbSuccess(new Error('connection Error'))
-      const request = makeApp()
-      const response = await request.get('/ping').expect(410)
-      expect(response.body).toHaveLength(2)
-      expect(response.body[0]).toEqual('Database Error')
-      expect(response.body[1]).toEqual('S3 Error')
-    })
-  })
 })
 
 describe('status route test', () => {
-  let routes
-
-  const makeApp = () => {
-    const app = express()
-    routes(app)
-    return supertest(app)
-  }
-
-  beforeEach(() => {
-    routes = require('.')
-  })
-
   it('returns success when app ok', async () => {
     const request = makeApp()
-    await request.get('/status').expect(200)
+    const result = await request.get('/status')
+    expect(result.res.text).toContain(`${goodSymbol} General`)
+    expect(result.res.text).toContain(`${goodSymbol} S3`)
+    expect(result.res.text).toContain(`${goodSymbol} Database`)
+    expect(result.res.text).toContain(`${badSymbol} SFTP`)
```

Comment:
I'm intrigued to know why the config for `unit-test` does not have SFTP disabled ? ... perhaps it should?

Reply:
Ah no it shouldn't .... this is used for an in-memory mock of the sftp - see  https://github.com/elifesciences/elife-xpub/blob/6b699062f89073d5ebc4b0d2df4a57e9dd7e4f75/packages/component-meca/server/meca/export.test.js#L35